.github/workflows/deploy.yml

```
@@ -35,6 +35,21 @@ jobs:
       cloudflare-zone-id: ${{ secrets.STAGING_CLOUDFLARE_ZONE_ID }}
       cloudflare-api-token: ${{ secrets.STAGING_CLOUDFLARE_API_TOKEN }}
 
+  warm-staging:
+    uses: ./.github/workflows/terraform.yml
+    with:
+      env: warm-staging
+      workspace: warm-staging
+      did: did:web:staging.warm.ipni.storacha.network
```

Per https://github.com/storacha/project-tracking/issues/454 this should be:
```suggestion
      did: did:web:staging.ipni.warm.storacha.network
```

Also please note the URL.

---

see the mini discussion in the comments of https://github.com/storacha/project-tracking/issues/454. I proposed going with `staging.warm.<service>.storacha.network` because that's more coherent with our current deployment scripts, which usually do `<environment>.<service>.storacha.network`.

We can of course go with the original proposal if you have strong opinions.

---

I DO!